examples/config_cpu_load_metrics.yaml

```diff
@@ -0,0 +1,28 @@
+extensions:
+  sumologic:
+    access_id: <my_access_id>
+    access_key: <my_access_key>
+    collector_name: <my_collector_name>
+
+# Define your receivers...
+receivers:
+  hostmetrics:
+    collection_interval: 30s
+    scrapers:
+      load:
+
+# Define your processors...
+processors:
+
+exporters:
+  sumologic:
+    auth:
+      authenticator: sumologic
```

Comment:
We can skip auth as we set it by default :)

Reply:
I removed this from example.